packages/govuk-frontend/src/govuk/components/tabs/tabs.mjs

```diff
@@ -64,6 +67,28 @@ export class Tabs extends GOVUKFrontendComponent {
     this.boundTabKeydown = this.onTabKeydown.bind(this)
     this.boundOnHashChange = this.onHashChange.bind(this)
 
+    const $tabList = this.$module.querySelector('.govuk-tabs__list')
+    const $tabListItems = this.$module.querySelectorAll(
+      'li.govuk-tabs__list-item'
+    )
+
+    if (!$tabList) {
+      throw new ElementError($tabList, {
+        componentName: 'Tabs',
+        identifier: '.govuk-tabs__list'
+      })
+    }
+
+    if (!$tabListItems.length) {
+      throw new ElementError(null, {
+        componentName: 'Tabs',
+        identifier: '.govuk-tabs__list-item'
+      })
+    }
+
+    this.$tabList = $tabList
+    this.$tabListItems = $tabListItems
```

Comment:
Love it, thanks @domoscargin

If you can add them as class fields too?

```mjs
  /** @private */
  $tabs

  /** @private */
  $tabList

  /** @private */
  $tabListItems
```

Won't need types as they can be inferred in the constructor

Reply:
Donezos